api/controllers/PrivateController.js

```diff
@@ -56,6 +57,23 @@ module.exports = {
       return res.json({ results })
     })
   },
+  sendSubsDue: function (req, res) {
+    var dbCall = queryString => cb => {
```

Comment:
I don't think this code (which is awesome) belongs in this function. It should set in this file at the top level and be used to refactor all the handlers